Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.h

```diff
@@ -96,6 +96,7 @@ class RemoteLayerTreeHost {
     RemoteLayerTreeDrawingAreaProxy* m_drawingArea { nullptr };
     WeakPtr<RemoteLayerTreeNode> m_rootNode;
     HashMap<WebCore::GraphicsLayer::PlatformLayerID, std::unique_ptr<RemoteLayerTreeNode>> m_nodes;
+    HashMap<WebCore::LayerHostingContextIdentifier, WebCore::GraphicsLayer::PlatformLayerID> m_hostingLayers;
```

Comment:
How long do entries stay in these two hashes? Is there a point where we can assert that they are empty?

Reply:
Until the layers are removed.  We can basically only assert that they are empty when we have no layers, which doesn't seem useful.